integration_tests/src/test/kotlin/integration/GameloopIT.kt

```diff
@@ -32,6 +34,7 @@ class GameloopIT {
 
     @Test
     fun iosGameloop() {
+        assumeFalse(isWindows)
```

Comment:
👍

Reply:
@adamfilipow92 it causes way too much conflicts. Technically it can atleast run it on windows. But the return and errors coming back make it a pain to fix.

Reply:
And after thinking about it. Most people working with IOS applications will not be attempting to have them running on windows.